samplePDF/samplePDFFDBase.cpp

```diff
@@ -788,6 +825,51 @@ void samplePDFFDBase::CalcXsecNormsBins(int iSample){
   return;
 }
 
+//LW 
+//Setup chosen oscillation calculator for each subsample
+//Default Baseline Implementation
+//Add your own implementation in experiment specific SamplePDF Class if necessary!!
+void samplePDFFDBase::SetupOscCalc(double PathLength, double Density)
```

Comment:
Can you add some comments throughout this function so it's clear why you're doing the different setups